Automattic/vip-go-mu-plugins
# Releasing Jetpack Connection Pilot by default to all sites

## Description

This PR removes the changes introduced in https://github.com/Automattic/vip-go-mu-plugins/pull/2197 to perform a staged rollout of Connection Pilot to all new sites.

Once this changes are merged, Connection Pilot will be enabled and will try to reconnect Jetpack to all sites.

## Changelog Description

### Releasing Jetpack Connection Pilot

We are now opening Jetpack Connection Pilot to all sites by default.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Check out PR.
2. Disconnect your site from Jetpack/create a new site.
3. After some minutes, your site should be connected to Jetpack automatically again.
